rmw_zenoh_cpp/config/DEFAULT_RMW_ZENOH_ROUTER_CONFIG.json5

```diff
@@ -69,9 +136,10 @@
   timestamping: {
     /// Whether data messages should be timestamped if not already.
     /// Accepts a single boolean value or different values for router, peer and client.
-    /// PublicationCache which is required for transient_local durability
-    /// only works when time-stamping is enabled.
-    enabled: { router: true, peer: true, client: false },
+    ///
+    /// ROS setting: PublicationCache which is required for transient_local durability
+    ///              only works when time-stamping is enabled.
+    enabled: { router: true, peer: true, client: true },
```

Comment:
It looks like this is the only difference from the default config.

Reply:
Yes: ROS is not using the "client" mode for now. But if one want to try, better to have time-stamping enabled for transient_local support (via PublicationCache and QueryingSubscriber).